lib/middleman/s3_metadata/version.rb

```diff
@@ -1,5 +1,5 @@
 module Middleman
   module S3Metadata
-    VERSION = '0.1.2'
+    VERSION = '0.1.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.